### PR TITLE
[MIG][11.0] asdcust: Migrate module to v11

### PR DESCRIPTION
**Pull request by luisg123v**
This commit migrates the project to v11. The following changes were
performed:
- Updated Odoo version from 10.0 -> 11.0
- Updated Python version from 2.7 -> 3.5
- Updated PostgreSQL version to 9.6
-Added missing new lines add end of file
- Removed encoding headers from all .py files

### CODE REVIEW

**Comment by luisg123v** [comment]
@ruiztulio 
Do you agree with these changes? If you do, could you create a new branch 11.0 in this project, so this PR can target it?

CC @umiphos

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ cache:
     - $HOME/.cache/pip
 
 addons:
+  postgresql: "9.5"
```
Comment (ruiztulio) [inline .travis.yml]: @luisg123v 9.6

**Review by ruiztulio — COMMENTED** [review]
@luisg123v The branch 11.0 is created now

File: .travis.yml (hunk 1 above)
Comment (luisg123v) [reply .travis.yml]: @ruiztulio 
Done. Also changed the base branch and edited the commit message to reflect the PostgreSQL version

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-[![Build Status](https://travis-ci.com/Vauxoo/asdcust.svg?token=rYtuFox23ptYNEopUvKo&branch=10.0)](https://travis-ci.com/Vauxoo/asdcust)
-[![Coverage Status](https://coveralls.io/repos/github/Vauxoo/asdcust/badge.svg?branch=10.0&t=4G2q8J)](https://coveralls.io/github/Vauxoo/asdcust?branch=10.0)
-[![Runbot Status](http://runbot.vauxoo.com/runbot/badge/93/10.0.svg)](http://runbot.vauxoo.com/runbot/repo/git-github-com-vauxoo-asdcust-git-93)
+[![Build Status](https://travis-ci.com/Vauxoo/asdcust.svg?token=rYtuFox23ptYNEopUvKo&branch=11.0)](https://travis-ci.com/Vauxoo/asdcust)
+[![Coverage Status](https://coveralls.io/repos/github/Vauxoo/asdcust/badge.svg?branch=11.0&t=4G2q8J)](https://coveralls.io/github/Vauxoo/asdcust?branch=11.0)
+[![Runbot Status](http://runbot.vauxoo.com/runbot/badge/93/11.0.svg)](http://runbot.vauxoo.com/runbot/repo/git-github-com-vauxoo-asdcust-git-93)
 [![Translation status](http://weblate.vauxoo.com/widgets/asdcust-10-0/-/svg-badge.svg)](http://weblate.vauxoo.com/engage/asdcust-10-0/?utm_source=widget)
```
Comment (ruiztulio) [inline README.md]: @luisg123v 11.0 too (we are not using it, but just in case)

Comment (luisg123v) [reply README.md]: Done! I had searched "10.0", but that was "10-0"

File: .travis.yml (hunk 1 above)
Comment (ruiztulio) [reply .travis.yml]: @luisg123v did you pushed the changes here? because I still see 9.5

Comment (luisg123v) [reply .travis.yml]: @ruiztulio 
You're right. I pushed to vauxoo-dev, as I use to do, but in this case it's a fork in my user.

**Pull request merged**